lib/python/bailoclient/client.py

```diff
@@ -396,9 +448,17 @@ def __increment_version(self, model_uuid: str):
         """
 
         model_versions = self.api.get(f"model/{model_uuid}/versions")
-        model_versions = [
-            int(model_version["version"]) for model_version in model_versions
-        ]
+        try:
+            model_versions = [
+                int(model_version["version"]) for model_version in model_versions
```

Comment:
This is a holdover from the old system where we enforced linear versioning from 1, 2, 3, 4, etc.